README.md

```diff
@@ -1,6 +1,25 @@
-# README 
+# README
 [![Maintainability](https://api.codeclimate.com/v1/badges/ce8783c839063c27b944/maintainability)](https://codeclimate.com/github/Fermented-Software/Kimchi/maintainability)
-[![Coverage Status](https://coveralls.io/repos/github/egydiopacheco/Kimchi/badge.svg?branch=main)](https://coveralls.io/github/egydiopacheco/Kimchi?branch=main)
+[![Coverage Status](https://coveralls.io/repos/github/Fermented-Software/Kimchi/badge.svg?branch=main)](https://coveralls.io/github/Fermented-Software/Kimchi?branch=main)
+![GA Release Workflow main](https://github.com/egydiopacheco/Kimchi/actions/workflows/ci-release.yaml/badge.svg?branch=main)
```

Comment:
Those 2 last badges are linked to my GH profile, it redirects to our organization but I don't know for how long...so it is better to change it

Reply:
Fix by 434d238c6ab2ec324fbbbb42cc2688211939dd82